install_config/cluster_metrics.adoc

```diff
@@ -162,71 +172,77 @@ to run cluster metrics with non-persistent data, running with non-persistent
+=== Using Secrets
 
-By default, the *metrics deployer* auto-generates self-signed certificates for
-use between components. Since these are self-signed certificates they will not
-be automatically trusted by a web browser and it is advisable to use your own
-certificates for anything being accessed outside of the OpenShift cluster. This
-is especially important for the Hawkular Metrics server as it needs to be accessible
-in a browser for the console to function.
+By default, the Metrics Deployer auto-generates self-signed certificates for
+use between components. Because these are self-signed certificates, they are not
+automatically trusted by a web browser; therefore, it is advisable to use your
+own certificates for anything being accessed outside of the OpenShift cluster.
+This is especially important for the Hawkular Metrics server as it must be
+accessible in a browser for the web console to function.
 
-To provide your own certificates and replace the internally used ones, you
-can pass these values as link:../dev_guide/secrets.html[secrets] to the
-*metrics deployer*.
+The Metrics Deployer requires that you manually create a *metrics-deployer*
+secret whether you are link:#metrics-using-secrets-byo-certs[providing your own
+certificates] or link:#metrics-using-secrets-autogenerated[using generated
+self-signed certificates].
 
-Alternatively you can instead use a
-link:../install_config/cluster_metrics.html#reencrypting-route[re-encrypting route] which
-will allow the self-signed certificates to remain used internally while allowing your own
-certificates to be used for externally access. If you wish to use a re-encrypting route
-you should not set the certificates as a secret.
+[[metrics-using-secrets-byo-certs]]
+==== Providing Your Own Certificates
 
+To provide your own certificates and replace the internally used ones, you
+can pass these values as link:../dev_guide/secrets.html[secrets] to the
+Metrics Deployer.
 
-[WARNING]
+[NOTE]
 ====
-Setting the value via secrets will *replace* the internally used certificates and as such these
-certificates *must* be valid for both the externally used hostnames as well as the externally hostname. For
-`hawkular-metrics` this means the certificate must be value if *'hawkular-metrics'* as well
-as the value specified in *HAWKULAR_METRICS_HOSTNAME*.
-
-If you are unable to add the internal hostname to your certificate, then you will need to a
-link:../install_config/cluster_metrics.html#reencrypting-route[re-encrypting route].
+Alternatively, you can use a
+link:../install_config/cluster_metrics.html#metrics-reencrypting-route[re-encrypting
+route] which allow the self-signed certificates to remain in use internally
+while allowing your own certificates to be used for externally access. If you
+want to use a re-encrypting route, you must not set the certificates as a
+secret, but you must still link:#metrics-using-secrets-autogenerated[create a
+"dummy" secret] for the Metrics Deployer.
```

Comment:
This part is not exactly correct:
>but you must still link:#metrics-using-secrets-autogenerated[create a "dummy" secret] for the Metrics Deployer.

You need a secret to be created, but it does not have to be a dummy secret. The re-encrypting route only affects the Hawkular Metrics certificate and secrets can handle more than that.